frontend/public/components/catalog/_catalog.scss

```diff
@@ -181,7 +181,7 @@ $catalog-tile-width: $co-m-catalog-tile-width;
   }
 
   &__overlay--right {
-    position: fixed !important;
+    position: absolute !important;
```

Comment:
Shouldn't this be applied as an override to all the modals that are using PF modal wrapper?

Reply:
updated